Web Ui/src/sections/TDDCycles-Visualization/components/TDDChart.tsx

```diff
@@ -142,8 +146,10 @@ function TDDCharts(commits: CommitDataObject[] | null,  jobsByCommit: JobDataObj
 
     return (
         <>
-            <h1>Gráfico de Lineas y puntos</h1>
-            <Line height="100" data={dataLineChart} options={optionsLineChart} onClick={onClick} ref={chartRef}/>
+            <div className="lineChartContainer">
```

Comment:
instead of adding this div you could return the div <>div...</> ---->  <div classname="">...</>

Reply:
<div classname="lineChartContainer"> </..>

Reply:
I fixed it now, please check it